### PR TITLE
Added README

### PR DESCRIPTION
**Pull request by tynsh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+```
+export GOROOT=$HOME/go
+export PATH=$PATH:$GOROOT/bin
+go get -insecure -u github.com/nnev/kasse`
+cd ~/go/src/github.com/nnev/kasse && sqlite3 kasse.sqlite < schema.sql
+```
+
+## Testing
+`kasse --hardware=false`
```
Comment (Merovius) [inline README.md]: Only one -

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+## Kasse
+
+This is an Implementation of a payment system for the
+[Heidelberg Chaostreff](https://www.noname-ev.de). It is currently in a basic
+stage of devellopment, so this README.md should give you enough information to
```
Comment (Merovius) [inline README.md]: s/devellopment/development/

Comment (joker234) [inline README.md]: devellopment → development

Comment (Merovius) [inline README.md]: "so this README.md should give you enough information to install it and start testing" is a weird implication in my mind. Maybe there is an "only" missing here?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+## Installing
+
+Currently you need to install libnfc to build kasse. This will not be needed in
+the future. In Debian you need to install the following packages:
+
+- libnfc-dev
+- sqlite3
+- golang
+
+
```
Comment (Merovius) [inline README.md]: Some comment seems in order here. Like "The following will build the code and give you a basic environment to run kasse".

File: README.md (hunk 1 above)
Comment (Merovius) [inline README.md]: It is important, that the binary runs in the path containing kasse.sqlite from above or gives the full path to it with -connect.

**Comment by Merovius** [comment]
Please don't merge this, before the branch is rebased, such that the merge commit is gone. I can show tobias how to do that offline.

**Pull request merged**